src/makeRelativeDate.README.md

```diff
@@ -0,0 +1,23 @@
+
+# `makeRelativeDate`
+
+Takes an absolute date in epoch format and returns a string representing relative time ago.
+
+## Props
```

Comment:
we're trying to use a common way of describing methods and props in the readme (largely borrowed from most of Sindre Sorhus's readmes) see https://github.com/guardian/libs/blob/main/src/logger.README.md#methods for a good example